Equinox/GameObject.cpp

```diff
@@ -0,0 +1,73 @@
+﻿#include "GameObject.h"
+#include "BaseComponent.h"
+#include "Globals.h"
+#include <GL/glew.h>
+
+GameObject::GameObject()
+{
+}
+
+GameObject::~GameObject()
+{
+}
+
+void GameObject::SetParent(GameObject* new_parent)
+{
+	_parent = new_parent;
+}
+
+GameObject* GameObject::GetParent()
+{
+	return _parent;
+}
+
+std::vector<GameObject*> GameObject::GetChilds()
+{
+	return _childs;
+}
+
+void GameObject::AddChild(GameObject* child)
+{
+	if (child != nullptr)
+	{
+		_childs.push_back(child);
```

Comment:
Set the parent of the child and remember to disassociate the child with its previous parent